resources/views/admin/internet/mac_addresses/list.blade.php

```diff
@@ -0,0 +1,125 @@
+<div class="card">
+    <div class="card-header">@lang('internet.admin.manage_register_devices')</div>
+    <div class="card-body">
+        <form action="{{ route('internet.mac_addresses.add') }}" method="post">
+            @if ($errors->any())
+                <div class="alert alert-danger">
+                    <ul>
+                        @foreach ($errors->all() as $error)
+                            <li>{{ $error }}</li>
+                        @endforeach
+                    </ul>
+                </div>
+            @endif
+            <div class="form-row align-items-center">
+                @csrf
+                <div class="col-auto">
+                    <label for="inlineFormInput">@lang('internet.user_id')</label> {{-- TODO: Better user selection --}}
+                    <input type="text" class="form-control mb-2" name="user_id" placeholder="1">
+                </div>
+                <div class="col-auto">
+                    <label for="inlineFormInput">@lang('internet.mac_address')</label>
+                    <input type="text" class="form-control mb-2" name="mac_address" placeholder="00:00:00:00:00:00">
+                </div>
+                <div class="col-auto">
+                    <label for="inlineFormInput">@lang('internet.comment')</label>
+                    <input type="text" class="form-control mb-2" name="comment" placeholder="Laptop">
```

Comment:
Shouldn't we use `@lang(...)` here for the placeholder?

Reply:
(Same in english.)[https://4cdn.hu/kraken/image/upload/s--QORejLZt--/c_limit,w_1160/6r1HJ7w7OKEGX2Jos.jpeg] :D